challenges/src/main/java/hashtable/Hashtable.java

```diff
@@ -0,0 +1,49 @@
+package hashtable;
+
+
+public class Hashtable {
+
+    int size;
+    String[] map;
```

Comment:
Should be array of linked lists of a "storage" class. So something like;
List<Storage>[] map;

Where Storage has a key/value pair.